Source/WebCore/page/Page.cpp

```diff
@@ -4224,6 +4210,20 @@ RefPtr<const LocalFrame> Page::localMainFrame() const
     return dynamicDowncast<LocalFrame>(mainFrame());
 }
 
+RefPtr<Document> Page::localTopDocument()
```

Comment:
This isn't needed if there's a const version that returns the exact same thing.  I'd expect the other to return a RefPtr<const Document> or something if this is here.